core/ez/__init__.py

```diff
@@ -400,13 +418,17 @@ def _setup(__ez=_EZ.ez):
 
     del Modules
 
-    from ez.events import Plugins
+    from ez.plugins import PluginEvent, __pm
 
-    emit(Plugins.WillLoad)
-    load_plugins()
-    emit(Plugins.DidLoad)
+    emit(PluginEvent.WillLoad)
+    # TODO: WTF????
+    __pm.load_plugins(
+        "test-plugin",
+        "title-changer"
+    )
+    emit(PluginEvent.DidLoad)
```

Comment:
`iterdir`?

Reply:
No, it should pull the data from the database

Reply:
Ok makes sense